wasm2c/wasm-rt-impl.c

```diff
@@ -417,6 +417,8 @@ const char* wasm_rt_strerror(wasm_rt_trap_t trap) {
       return "Invalid call_indirect";
     case WASM_RT_TRAP_UNCAUGHT_EXCEPTION:
       return "Uncaught exception";
+    case WASM_RT_TRAP_UNALIGNED:
```

Comment:
I don't *think* we need a new runtime trap because I believe this is checked at validation-time? But I might be confused -- do the tests ever raise this or test for this trap?

Reply:
it seems like this is a runtime trap, and so I think we may require this